beagle/src/main/kotlin/br/com/zup/beagle/android/context/Bind.kt

```diff
@@ -23,7 +23,11 @@ import br.com.zup.beagle.android.context.tokenizer.TokenParser
 import br.com.zup.beagle.android.utils.BeagleRegex
 import br.com.zup.beagle.android.utils.getExpressions
 import br.com.zup.beagle.core.BeagleJson
+import com.squareup.moshi.rawType
```

Comment:
<table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-android/blob/7dc5143a20bba18191665b85d628b76cd15527b0/beagle/src/main/kotlin/br/com/zup/beagle/android/context/Bind.kt#L26"></span>The import 'com.squareup.moshi.rawType' is unused.</td>
    </tr>
  </tbody>
</table>